### ISSUE TITLE
Why? This already exists

### ISSUE THREAD

**Issue by Ebonsignori** [saibotsivad/obsidian-at-people#7]
I'd be happy if you did this before I created https://github.com/Ebonsignori/obsidian-at-symbol-linking to do essentially the same thing, but from your commit history and issue history it looks like you are copying some of my logic and sourcing my issues for features. This is all open source, why do this?

**Comment by Ebonsignori** [comment]
Ah I see your reply to the thread, https://forum.obsidian.md/t/mention-support-for-people-identities/6163/71?u=ebonsignori that was a bit underhanded. Mine was waiting in PR just as yours was

**Comment by Ebonsignori** [comment]
If an open source project already exists the courteous thing to do it [open a PR to contribute](https://github.com/alangrainger/obsidian-google-photos/pull/18) not race to steal credit of others' work

**Comment by Ebonsignori** [comment]
If you're wanting more open source dev work to do for fun, this plugin is highly in demand, but seems like it might be abandoned and is rather buggy at times and the source code is highly obfuscated https://github.com/uphy/obsidian-reminder

**Comment by saibotsivad** [comment]
Yeah, I was also trying to figure out a way to somehow manage and merge `EditorSuggest` instances, which is wildly hacky. I was really hoping to not get into custom HTML, but that might be the way to go until Obsidian supports this better 😬 

Don't know if you saw elsewhere, but I had raised a [discussion on the Tasks plugin](https://github.com/obsidian-tasks-group/obsidian-tasks/discussions/2236), to see if anyone had ideas.

Basically the only option right now (for Tasks anyway) is relying on load order: load the Tasks plugin last, so its instance of `EditorSuggest` doesn't collide.
